### DISCUSSION TITLE
Database Elasticsearch messages flood

### DISCUSSION THREAD

**Discussion by sleepingbel** [Security-Onion-Solutions/securityonion#6693] [Unsupported Versions] [answered]
Hello,

SO standalone iso install 2.3.90 updated to 2.3.91 Hotfix 
OnPrem
ZEEK metadata

8 vCPU
24 GB vRAM
1 TB diskspace

I have seen that in 24h over ±60000 logs are created for elasticsearch database with following message:

"Loading the fielddata on the _id field is deprecated and will be removed in future versions. If you require sorting or aggregating on this field you should also include the id in the body of your documents, and map this field as a keyword field that has [doc_values] enabled"

How can i stop this?

Regards
Bart

**Comment by dougburks** [comment, marked as answer]
We'll take a look for the next release:
https://github.com/Security-Onion-Solutions/securityonion/issues/6703
